### DISCUSSION TITLE
Building a rust based R package on r-universe

### DISCUSSION THREAD

**Discussion by JosiahParry** [r-universe-org/help#269] [General]
I have been attempting to build an extendr based R package on r-universe but have been unable to do so. The [action fails](https://github.com/r-universe/josiahparry/actions/runs/5489189503/jobs/10005749840) with the message: 

```
  error: rustup could not choose a version of cargo to run, because one wasn't specified explicitly, and no default is configured.
  help: run 'rustup default stable' to download the latest stable release of Rust and set it as your default toolchain.
```

Is this something that should be addressed in the Makevars or on the r-universe side?

**Comment by JosiahParry** [comment]
Perhaps related to https://github.com/r-universe-org/help/issues/268

**Comment by jeroen** [comment]
Yes we were trying to upgrade rust. Should be fixed now!

**Reply by JosiahParry** [reply]
Brilliant! It works :) Thank you <3
